src/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -42,6 +42,18 @@ public abstract class GenericLoopbackServer : IDisposable
         public abstract Task AcceptConnectionAsync(Func<GenericLoopbackConnection, Task> funcAsync);
 
         public abstract void Dispose();
+
+        public async Task<HttpRequestData> AcceptConnectionSendResponseAndCloseAsync(HttpStatusCode statusCode = HttpStatusCode.OK, string content = "")
```

Comment:
How is this different than HandleRequestAsync?

Reply:
I missed the similarity. I don't know why two functions existed in LoopBack server. AcceptConnectionSendResponseAndCloseAsync() seems more descriptive (per earlier @krwq comment) so I left it that calling HandleRequestAsync().